src/main/frontend/components/conversion.cljs

```diff
@@ -145,18 +145,18 @@
           [:div ;; Normal UX stage 2: close stage 1 UI, show the action description as admolition
            (if (and (= @*solid-format :triple-lowbar)
                     (= @*dir-format :legacy))
-             (ui/admonition :tip [:p (t :file-rn/need-action)])
-             [:p (t :file-rn/need-action)])
+             (ui/admonition :tip [:p (t ::need-action)])
+             [:p (t ::need-action)])
            [:p
             (ui/button
-             (str (t :file-rn/all-action) " (" (count rename-items) ")")
+             (str (t ::all-action) " (" (count rename-items) ")")
              :on-click <rename-all
              :class "text-md p-2 mr-1")
-            (t :file-rn/or-select-actions)
+            (t ::or-select-actions)
             [:a {:on-click <close-modal-on-done}
-             (t :file-rn/close-panel)]
-            (t :file-rn/or-select-actions-2)]
-           [:p (t :file-rn/legend)]
+             (t ::close-panel)]
+            (t ::or-select-actions-2)]
+           [:p (t ::legend "🟢" "🟡" "🔴")]
```

Comment:
Icons and symbols shouldn't be part of the translatable string. We shouldn't have to invalidate the translations to change the icons.